pkg/controllers/webhook/controller.go

```diff
@@ -777,7 +892,10 @@ func (c *controller) buildResourceValidatingWebhookConfiguration(ctx context.Con
 					}
 				}
 			}
+			rules := p.GetSpec().Rules
+			operationStatusMap = computeOperationsForValidatingWebhookConf(rules, operationStatusMap)
```

Comment:
Do we need to filter by rule types? This [line](https://github.com/kyverno/kyverno/pull/8437/files#diff-6c0dd4f85a678b8261eb41e6ab885470b2d6633cb9d5728bb31dae8e532ef85fR887) selects all rule types though.

Reply:
I dont think we need to.

Reply:
For validate webhook which may be fine as we configure it by all types of rules.